orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/WaitForCapacityMatchTask.groovy

```diff
@@ -58,8 +58,15 @@ class WaitForCapacityMatchTask extends AbstractInstancesCheckTask {
         desired = ((Map<String, Integer>)serverGroup.capacity).desired
       }
 
-      splainer.add("checking if capacity matches (desired=${desired}, instances.size()=${instances.size()}) ")
-      if (desired != instances.size()) {
+      splainer.add("checking if capacity matches (desired=${desired}, target=${stage.context.targetDesiredSize ?: "none"} instances.size()=${instances.size()}) ")
```

Comment:
I am missing something, where is the `targetDesiredSize` computed to take `targetHealthyDeployPercentage` into account?

Reply:
It is computed [here](https://github.com/spinnaker/orca/blob/master/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/WaitForCapacityMatchTask.groovy#L38)

Reply:
got it, thanks! I think it looks reasonable. But i would get @dreynaud input as he is the ultimate expert here

Reply:
I'm a bit rusty on this but the explanation makes sense and I don't see issues 👍

Reply:
nit: `instances.size()` loses meaning when viewed just as a log message: Anyone unfamiliar with the code would have to code dive to figure out what `instances.size()` is referring to - is it the desired or target?

Reply:
`instances.size()` is meaningful to me, it is the current number of instances. As in, "your desired is X, your target is Y, but you currently have Z"

Reply:
I can update that to `current` if that makes it easier to read.